Mathlib/Data/Int/Init.lean

```diff
@@ -292,6 +292,58 @@ lemma strongRec_of_lt (hn : n < m) : m.strongRec lt ge n = lt n hn := dif_pos _
 
 end strongRec
 
+/-! ### neg -/
+
+-- We want to use this lemma earlier than the lemmas simp can prove them with
+
+@[simp, nolint simpNF] protected lemma neg_le_neg_iff : -a ≤ -b ↔ b ≤ a where
+  mp := Int.le_of_neg_le_neg
+  mpr := Int.neg_le_neg
```

Comment:
This is in Lean already. Can we wait to merge this PR for a day or two, so we can move to the `v4.19.0-rc1` toolchain, and then clean up this PR, rather than me having to clean it up afterwards? :-)

Reply:
Sure, no problem with me

Reply:
It is.